### PR TITLE
Increased chain support

### PR DESCRIPTION
**Pull request by mihoward21**
- update url for zora testnet subgraph (old one stopped working, they must have got rid of it)
- fix the symbols for native currencies on networks other than eth & matic
- fix some old tests for chain support
- allow empty (or partial) config to be passed into `useSplitsClient`. Only overwrite a value if it's key is included in the passed in config. That way if you just want to pass in an empty config it won't overwrite anything. But if you want to clear out a value (i.e. get rid of a wallet client cause a user disconnected) you can pass in { walletClient: undefined } and it will clear it out.
- support all of our chains in storybook (not just the ones we have an alchemy rpc for)
- if it's a non-alchemy chain, pass in an empty erc20 list to the earnings call. eventually we should expose that as a param, for now it allows us to load the native token on whatever chain if it's not an alchemy url.

https://linear.app/splits/issue/PE-1631/use-provider-values-when-no-config-passed-in
https://linear.app/splits/issue/PE-1814/fix-native-token-symbols-in-sdk
https://linear.app/splits/issue/PE-1775/verify-splitskit-works-with-aurora-fantom-avalanche

### CODE REVIEW

File: packages/splits-kit/src/components/DisplaySplit/index.tsx [hunk 1]
```diff
@@ -38,11 +44,15 @@ const DisplaySplit = ({
     isLoading: isLoadingMetadata,
   } = useSplitMetadata(address)
 
+  const includeActiveBalances = true
+  const erc20TokenList = isAlchemyChainId(chainId)
```
Comment (mihoward21) [inline packages/splits-kit/src/components/DisplaySplit/index.tsx:48]: you know...this should probably use the same function that the core sdk uses. let me see if i can make that change. then don't need to define this here.

File: packages/splits-kit/src/constants/chains.ts [hunk 2]
```diff
@@ -82,7 +87,7 @@ export const CHAIN_INFO: ChainInfo = {
     label: 'Goerli',
     logoUrl: '/networks/ethereum_logo.svg',
     rpcUrls: [
-      `https://eth-goerli.g.alchemy.com/v2/${process.env.STORYBOOK_GOERLI_ALCHEMY_API_KEY}`,
+      `https://eth-goerli.g.alchemy.com/v2/${process.env.STORYBOOK_ALCHEMY_API_KEY}`,
```
Comment (mihoward21) [inline packages/splits-kit/src/constants/chains.ts:90]: shouldn't these rpc urls be defined in storybook, but not in splits kit? like they're just for our usage in storybook. i guess i'm a bit confused as to what's part of storybook and what's part of the splits-kit package that others are using.

Comment (shrimpynuts) [reply packages/splits-kit/src/constants/chains.ts:85]: hm, yeah you're right, these are only used in the storybook, so I set this up a bit strange..

perhaps we just want separate constant mapping for these rpcUrls within the storybook directory, so these don't get included in the bundle?

should probably have a .env.example as well

**Comment by mihoward21** [comment]
ok all ready for review now i think

**Review by shrimpynuts — APPROVED** [review]
lgtm, just may want to move those rpcUrls into storybook?

File: packages/splits-kit/src/constants/chains.ts (hunk 2 above)
Comment (mihoward21) [reply packages/splits-kit/src/constants/chains.ts:85]: ok yea lemme do a little refactoring

**Pull request merged**